src/SFA.DAS.Tools.Servicebus.Support.Application/Queue/Queries/GetUserSession/GetUserSessionQueryResponse.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using SFA.DAS.Tools.Servicebus.Support.Domain;
 using SFA.DAS.Tools.Servicebus.Support.Domain.Queue;
 
 namespace SFA.DAS.Tools.Servicebus.Support.Application.Queue.Queries.GetUserSession
 {
     public class GetUserSessionQueryResponse
-    {
-        public bool UserHasExistingSession { get; set; }
+    {        
+        public UserSession UserSession { get; set; }
```

Comment:
Add a method that returns the result of the null check such as:
`public bool IsValid() => UserSession != null;`

Reply:
Done